.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2023 Linaro Ltd.
+#
+name: "Builds"
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run periodically to check that it still compiles
+    - cron: '13 13 * * 1'
+  workflow_dispatch:
+
+jobs:
+  job:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - debian:testing
+          - debian:bookworm
+            #- debian:bullseye
+            #- debian:buster
+          - ubuntu:lunar
+          - ubuntu:jammy
+            #- ubuntu:focal
+            #- ubuntu:bionic
+            #- ubuntu:xenial
+        target:
+          - native
+          - aarch64-linux-gnu
+          - arm-linux-gnueabihf
+
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+
+      - name: Install meson
+        run: |
+          apt update
+          apt -y install --no-install-recommends meson build-essential
+
+      - name: Install cross-compilers
+        run: |
+          if [ ${{ matrix.target }} != native ]
+          then
```

Rather than having this a bash expression, you should use an `if:` on the yaml level so that you don't have unnecessary nesting here and can nicely see the step being skipped in the CI output:

```suggestion
        if:${{ matrix.target }} != 'native'
        run: |
```

---

ack